test-resources-build-tools/src/main/java/io/micronaut/testresources/buildtools/ServerUtils.java

```diff
@@ -259,13 +259,27 @@ public static void stopServer(Path serverSettingsDirectory) throws IOException {
     /**
      * Returns the default path to the settings directory for the test
      * resources server in case it needs to be shared between builds.
+     * Equivalent to calling {@link #getDefaultSharedSettingsPath(String)}
+     * without a namespace.
      *
      * @return the default path to the settings directory
      */
     public static Path getDefaultSharedSettingsPath() {
         return Paths.get(System.getProperty("user.home"), ".micronaut/test-resources");
```

Comment:
Should this just call `getDefaultSharedSettingsPath(null)`?

Reply:
ah sure that was the intent :facepalm: